### ISSUE TITLE
performance measure ???

### ISSUE THREAD

**Issue by peternara** [meijieru/crnn.pytorch#77]
hi, 
* Is the evaluation method shown in this paper("An End-to-End Trainable Neural Network for Image-based Sequence Recognition and Its Application to Scene Text Recognition") as follows?
  * paper results > III5K 78.2%, SVT, 80.8%, IC03 89.4%, IC13 86.7 (all None)

**perfect word match ?**

or

**character accuracy?**

or
**WAcc = 1-WER**
https://en.wikipedia.org/wiki/Word_error_rate

or
![image](https://user-images.githubusercontent.com/6295576/33062196-cdeeba08-cee1-11e7-9fb1-85f1f79db384.png)

or **Normalized Edit Distance & Edit Distance**  ?

or **others ???**

There is no special information in the paper, and I do not understand. What exactly is it? please let me know?

thanks~:)
